extensions/wikia/WikiaHubsServices/modules/WikiaHubsModuleWAMService.class.php

```diff
@@ -63,119 +63,17 @@ public function loadData($model, $params) {
-						'top_1k_days' => '431',
-						'top_1k_weeks' => '62',
-						'first_peak' => '2012-04-21',
-						'last_peak' => '2013-02-18',
-						'title' => 'League of Legends Wiki',
-						'url' => 'leagueoflegends.wikia.com',
-						'hub_id' => '2',
-						'wam_change' => '0.0039',
-						'admins' => [],
-						'wiki_image' => ImagesService::overrideThumbnailFormat('http://images4.wikia.nocookie.net/__cb20120828154214/wikiaglobal/images/thumb/e/ea/Wikia-Visualization-Main%2Cleagueoflegends.png/150px-Wikia-Visualization-Main%2Cleagueoflegends.png.jpeg', ImagesService::EXT_JPG),
-					],
-					1706 => [
-						'wiki_id' => '1706',
-						'wam'=> '99.7942',
-						'wam_rank' => '4',
-						'hub_wam_rank' => '3',
-						'peak_wam_rank' => '1',
-						'peak_hub_wam_rank' => '1',
-						'top_1k_days' => '431',
-						'top_1k_weeks' => '62',
-						'first_peak' => '2012-01-01',
-						'last_peak' => '2013-02-13',
-						'title' => 'Elder Scrolls',
-						'url' => 'elderscrolls.wikia.com',
-						'hub_id' => '2',
-						'wam_change' => '-0.0016',
-						'admins' => [],
-						'wiki_image' => ImagesService::overrideThumbnailFormat('http://images1.wikia.nocookie.net/__cb20121214183339/wikiaglobal/images/thumb/d/d4/Wikia-Visualization-Main%2Celderscrolls.png/150px-Wikia-Visualization-Main%2Celderscrolls.png', ImagesService::EXT_JPG),
-					],
-					3035 => [
-						'wiki_id' => '3035',
-						'wam'=> '99.6520',
-						'wam_rank' => '9',
-						'hub_wam_rank' => '4',
-						'peak_wam_rank' => '4',
-						'peak_hub_wam_rank' => '3',
-						'top_1k_days' => '431',
-						'top_1k_weeks' => '62',
-						'first_peak' => '2012-01-02',
-						'last_peak' => '2013-09-11',
-						'title' => 'Fallout Wiki',
-						'url' => 'fallout.wikia.com',
-						'hub_id' => '2',
-						'wam_change' => '0.0091',
-						'admins' => [],
-						'wiki_image' => null,
-					],
-				113 => [
-						'wiki_id' => '113',
-						'wam'=> '99.5000',
-						'wam_rank' => '17',
-						'hub_wam_rank' => '5',
-						'peak_wam_rank' => '2',
-						'peak_hub_wam_rank' => '2',
-						'top_1k_days' => '431',
-						'top_1k_weeks' => '62',
-						'first_peak' => '2012-05-04',
-						'last_peak' => '2013-05-07',
-						'title' => 'Memmory Alpha Wiki',
-						'url' => 'en.memory-alpha.org',
-						'hub_id' => '2',
-						'wam_change' => '-0.1000',
-						'admins' => [],
-						'wiki_image' => null,
-					],
-			]];
-
-			$data = [
-				'vertical_id' => $this->verticalId,
-				'api_response' => $apiResponse,
-			];
 
-			$structuredData = $this->getStructuredData($data);
-
-		} else {
-			$structuredData = WikiaDataAccess::cache(
-				$this->getMemcacheKey(
-					$lastTimestamp,
-					$this->skinName
-				),
-				6 * 60 * 60,
-				function () use( $model, $params ) {
-					return $this->loadStructuredData($model, $params);
-				}
-			);
-		}
+		$structuredData = WikiaDataAccess::cache(
+			$this->getMemcacheKey(
+				$lastTimestamp,
+				$this->skinName
+			),
+			6 * 60 * 60,
```

Comment:
I know it's not a new code, but maybe it could be stored in a var so it's traced more easily? :)